.github/actions/deploy-my-kibana/action.yml

```diff
@@ -82,7 +82,7 @@ runs:
 
         ### Oblt-cli user (Optional)
 
-        ${{ inputs.user }}
+        deploykibana
```

Comment:
Issue will still be assigned to the GitHub user that requested this action with the GitHub command